src/jenkins/DistributionManifest.groovy

```diff
@@ -0,0 +1,101 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+
+package jenkins
+
+
+class DistributionManifest implements Serializable {
```

Comment:
In Python we call this a bundle manifest, https://github.com/opensearch-project/opensearch-build/blob/main/src/manifests/bundle_manifest.py. I like "distribution manifest" a lot better, but let's be consistent? If you want to take on renaming everything "bundle" to "distribution" elsewhere that's A-OK too.

Reply:
Rename the Jenkins library to `BundleManifest` to be consistent with what we had for python.